data/speakers.yml

```diff
@@ -0,0 +1,809 @@
+    company: "Barclays"
+    URL: "https://www.barclays.com/"
+    intro: ""
+  - id: "brian_saltzman"
+    name: "Brian Saltzman (Horizon Asset)"
+    role: "Vice President of Development and Operations Engineering"
+    company: "Horizon Asset LLP "
+    URL: "https://www.horizon-asset.co.uk/"
+    intro: ""
+    photo: "speakers/brian_saltzman.jpeg"
+  - id: "michael_hrivnak"
+    name: "Michael Hrivnak (Red Hat)"
+    role: "Principal Software Engineer"
+    company: "Red Hat"
+    URL: "https://www.openshift.com/"
+    intro: ""
+    photo: "speakers/michael_hrivnak.jpeg"
+  - id: "matt_dorn"
+    name: "Matt Dorn (Red Hat)"
+    role: "Senior Software Engineer"
+    company: "Red Hat"
+    URL: "https://www.openshift.com/"
+    intro: ""
+    photo: "speakers/matt_dorn.jpeg"
+  - id: "duncan_lawie"
+    name: "Duncan Lawie (Credit Suisse)"
+    role: "Enterprise Architect - Data(base) and DevOps"
+    company: "Credit Suisse"
+    URL: "https://www.credit-suisse.com"
+    intro: ""
+    photo: "speakers/duncan_lawie.jpeg"
+  - id: "clive_grethe"
+    name: "Clive Grethe (ATOS)"
+    role: "Head of Cloud Center of Excellence and Application Transformation"
+    company: "ATOS"
+    URL: "https://www.atos.net"
+    intro: ""
+    photo: "speakers/clive_grethe.jpeg"
+  - id: "michael_sauter"
+    name: "Michael Sauter"
+    role: "Senior Backend Developer"
+    company: "Boehringer Ingelheim"
+    URL: "https://www.bix-digital.com"
+    intro: ""
+    photo: "speakers/michael_sauter.png"
+  - id: "redbeard"
+    name: "Brian Redbeard Harrington"
+    role: ""
+    company: "Red Hat"
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/redbeard.jpeg"
+  - id: "tejun_heo"
+    name: "Tejun Heo"
+    role: ""
+    company: "Facebook"
+    URL: "https://www.facebook.com"
+    intro: ""
+    photo: "speakers/tejun_heo.jpg"
+  - id: "filipe_brandenburger"
+    name: "Filipe Brandenburger"
+    role: ""
+    company: "Google"
+    URL: "https://www.Google.com"
+    intro: ""
+    photo: "speakers/filipe_brandenburger.jpeg"    
+  - id: "ben_breard"
+    name: "Ben Breard"
+    role: ""
+    company: "Red Hat"
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/ben_breard.jpeg"
+  - id: "annette_clewett"
+    name: "Annette Clewett"
+    role: ""
+    company: "Red Hat"
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/annette_clewett.jpeg"
+  - id: "travis_nielsen"
+    name: "Travis Nielsen"
+    role: ""
+    company: "Rook/Red Hat"
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/travis_nielsen.jpeg"   
+  - id: "mrunal_patel"
+    name: "Mrunal Patel"
+    role: ""
+    company: "Red Hat"
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/mrunal_patel.jpg	"
+  - id: "urvashi_mohnani"
+    name: "Urvashi Mohnani"
+    role: ""
+    company: "Red Hat"
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/urvashi_mohnani.jpeg	"
```

Comment:
@jiri-fiala Since this is your PR...would you mind being the one to clean up these extra spaces?